graph2tac/loader/predict_server.py

```diff
@@ -411,6 +451,7 @@ def prediction_loop(predict_server: PredictServer, context: GlobalContextMessage
                         log_cnts.n_predict += 1
 
                         # important line -- sending prediction
+                        predict_server.response_history.record_response(response)
```

Comment:
I thought you were going to write a wrapper for `send`. (Not saying you have to, just wondering why you didn't.)

Reply:
I was.  I got distracted (and it was a bit more complicated than I thought).  Let me see if I can do it easily.